src/SDK/Language/Swift.php

```diff
@@ -104,6 +104,11 @@ public function getIdentifierOverrides(): array
     public function getFiles(): array
     {
         return [
+            [
+                'scope'         => 'default',
+                'destination'   => '/Sources/{{ spec.title | caseUcfirst}}/OAuth/WebAuthComponent.swift',
+                'template'      => '/swift/Sources/OAuth/WebAuthComponent.swift.twig',
+            ],
```

Comment:
This file shouldn't be there for swift-server, if you generate for `server` mode there should be no error as `server` mode doesn't have any methods that will try to use the `WebAuthComponent`

Reply:
<img width="968" alt="Bildschirmfoto 2024-02-24 um 08 13 32" src="https://github.com/appwrite/sdk-generator/assets/46761404/e20d8d22-a6c2-446b-9a06-f7b83b96eb95">
<img width="968" alt="Bildschirmfoto 2024-02-24 um 08 13 43" src="https://github.com/appwrite/sdk-generator/assets/46761404/a1b73b36-95ba-40d3-ad8b-08803721017d">

As seen in the images above I produces a warning when you just pass the function argument. This warning can be silenced when you wrap in a closure.

Reply:
<img width="1728" alt="Bildschirmfoto 2024-02-24 um 08 16 27" src="https://github.com/appwrite/sdk-generator/assets/46761404/4b629fde-e652-4fed-9b08-d9c0960c5b55">

I get the following error in the server sdk if I don't include the file.

Reply:
It's because you've opened the server SDK, but [generated with client SDK mode](https://github.com/svenopdehipt/sdk-generator/blob/bf46d88ede82e9e8c38fbc6cad0fc2f4f9cc42bc/example.php#L40). You need to open `apple` instead of `swift-server`

Reply:
Ok, my bad. I have tested it again and now it works without the file.